zqd/handlers.go

```diff
@@ -55,6 +55,9 @@ func respond(c *Core, w http.ResponseWriter, r *http.Request, status int, body i
 
 func respondError(c *Core, w http.ResponseWriter, r *http.Request, err error) {
 	status, ae := errorResponse(err)
+	if status >= 500 {
+		c.requestLogger(r).Warn("Error", zap.Int("status", status), zap.Error(err))
```

Comment:
nit: suggest we keep error messages lower case:

```suggestion
		c.requestLogger(r).Warn("error", zap.Int("status", status), zap.Error(err))
```